app/screens/channel_members/index.js

```diff
@@ -13,10 +13,11 @@ import {getProfilesInChannel, searchProfiles} from 'mattermost-redux/actions/use
 import ChannelMembers from './channel_members';
 
 function mapStateToProps(state) {
+    const currentChannelMembers = getProfilesInCurrentChannel(state).filter((member) => member.delete_at === 0);
```

Comment:
This is going to cause this screen to re-render on every store change. We should either do this in a selector or in the component itself